packages/app-content-pages/src/shared/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link'
+import { arrayOf, bool, func, shape, string } from 'prop-types'
+import styled, { css } from 'styled-components'
+import { useTranslation } from 'next-i18next'
+import { Box, Button, Nav } from 'grommet'
+
+const StyledUl = styled.ul`
+  padding-inline-start: 0;
+`
+
+const StyledLi = styled.li`
+  list-style-type: none;
+  display: flex;
+`
+
+const StyledButton = styled(Button)`
+  text-decoration: none;
+  color: black;
+  padding: 5px 20px; // Same as Project About page sidebar
+  margin-bottom: 5px;
+  ${props =>
+    props.active &&
+    css`
+      background: #addde0; // accent-1
+      font-weight: bold;
+    `}
+
+  &:hover {
+    font-weight: bold;
+  }
+`
+
+const DEFAULT_HANDLER = () => {}
+
+function Sidebar({
+  activeSection = '',
+  className = '',
+  ariaLabel = '',
+  sections = [],
+  setActiveSection = DEFAULT_HANDLER
+}) {
+  const { t } = useTranslation('components')
+
+  return (
+    // <Nav aria-label={t('Publications.sideBarLabel')} className={className}>
+    <Nav aria-label={ariaLabel} className={className}>
+      <StyledUl>
+        {sections.map(section => (
+          <StyledLi key={section.name}>
+            <StyledButton
+              as={Link}
+              active={section.slug === activeSection}
```

Comment:
```suggestion
              active={section.slug === activeSection}
              aria-current = section.slug === activeSection ? 'true' : 'false'
```
`aria-current` here would provide a hint to screen readers as to which link is active.

Reply:
Good point - thanks!